config/locales/de.application_letters.yml

```diff
@@ -16,7 +16,7 @@ de:
       Programmierkenntnisse sind keine  Voraussetzung zur Teilnahme. Diese Information
       wird zur Gruppeneinteilung benötigt."
     login_before_creation: "Du musst angemeldet sein, um dich für Workshops bewerben zu können."
-    successful_update: "Bewerbung erfolgreich bearbeitet"
+    successful_update: "Die Bewerbung wurde erfolgreich aktualisiert"
```

Comment:
Maybe it seems inconsistent if there is an full sentence at this point and on "successful_deletion" and "successful_creation" there are just word groups